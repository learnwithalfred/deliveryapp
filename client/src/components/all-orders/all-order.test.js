import React from "react";
import { AllApiOrders } from "./all-orders.component";
import { shallow } from "enzyme";

describe("AllApiOrders component render", () => {
  let wrapper;
  let socketSendOrderUpdate = jest.fn();
  beforeEach(() => {
    const mockProps = {
      mocksocketSendOrderUpdate: socketSendOrderUpdate,
    };
    wrapper = shallow(<AllApiOrders {...mockProps} />);
  });

  it("should render AllApiOrders component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
