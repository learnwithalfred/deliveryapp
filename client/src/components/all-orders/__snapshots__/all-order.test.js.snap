// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllApiOrders component render should render AllApiOrders component 1`] = `ShallowWrapper {}`;
